### ISSUE TITLE
The content is outdated

### ISSUE THREAD

**Issue by hiepdang-ml** [d2l-ai/d2l-en#2595]
I found the book having very good content for the topics it covers. But the book stopped at GANs. Many not-very-new topics like YOLO, Diffusion were never discussed. I've seen some opened issues mentioned this several years ago but it seems no contents have been added. Will the book continue to be updated or it's archived?

**Comment by EricThomson** [comment]
I’ve chatted with some of the devs it is no longer being supported by Amazon which is really unfortunate as this was such an amazing project.  I’m not sure what happened it seems once the book was done the vision or funding or passion for a living supported resource dissipated. 

I’d really be interested to hear what happened!

I thought about forking and offering active support. I asked them about this and they said they would be fine if I did it. Unfortunately this would require a small team and active financial support there is no way I could pull it off as a side project it’s just too big.

**Comment by artofcomputing** [comment]
> I’ve chatted with some of the devs it is no longer being supported by Amazon which is really unfortunate as this was such an amazing project. I’m not sure what happened it seems once the book was done the vision or funding or passion for a living supported resource dissipated.

Most likely funding. To document and maintain content of this size requires substantial capital, because let's be real: **capital matters**.

> I thought about forking and offering active support. I asked them about this and they said they would be fine if I did it. Unfortunately this would require a small team and active financial support there is no way I could pull it off as a side project it’s just too big.

Is there anything stopping you from:
- Rounding supporters, maintainers, researchers, and users.
- Mapping the labor required to bring this book up to date.
- Analyzing all possible sources of funding (enterprise funds, user donations, advertisements, etc) based on the mapping.
- Publishing the results and the steps required to provide active support.

This is just a simplified process with absolutes here and there for brevity. It doesn't seem impossible to make it happen.

**Comment by EricThomson** [comment]
You are right it basically needs someone with the time and energy to do all that. That's not me right now. Who will be the hero? 😄

**Comment by artofcomputing** [comment]
> You are right it basically needs someone with the time and energy to do all that. That's not me right now. Who will be the hero? 😄

That's understandable. That hero ain't gonna be me either. Nothing personal, I simply won't champion a project of this size without any kind of guaranteed funding to cover my labor costs.
